.github/workflows/image.yaml

```diff
@@ -12,4 +12,11 @@ jobs:
       - uses: actions/checkout@v4
       - name: Build image
         run: |
-          docker build -t fms-hf-tuning:dev . -f build/Dockerfile
\ No newline at end of file
+          docker build -t fms-hf-tuning:dev . -f build/Dockerfile
+      - name: Free disk space
```

Comment:
Needs to run before the docker build otherwise it fails, so as a prereq to make sure the space is free before building the image